code/modules/projectiles/guns/energy.dm

```diff
@@ -731,20 +793,29 @@
 	flags_gun_features = GUN_CAN_POINTBLANK|GUN_ENERGY|GUN_AMMO_COUNTER|GUN_NO_PITCH_SHIFT_NEAR_EMPTY|GUN_AMMO_COUNT_BY_SHOTS_REMAINING
 	attachable_offset = list("muzzle_x" = 41, "muzzle_y" = 15,"rail_x" = 22, "rail_y" = 24, "under_x" = 30, "under_y" = 8, "stock_x" = 22, "stock_y" = 12)
 
-	aim_slowdown = 1
-	wield_delay = 1.5 SECONDS
+	aim_slowdown = 0.7
+	wield_delay = 0.8 SECONDS
 	scatter = 1
 	fire_delay = 0.2 SECONDS
+	burst_delay = 0.25 SECONDS
 	accuracy_mult = 1
 	accuracy_mult_unwielded = 0.3
 	scatter_unwielded = 30
+	movement_acc_penalty_mult = 6
 	damage_falloff_mult = 0.3
+	windup_sound = 'sound/weapons/guns/fire/tank_minigun_start.ogg'
 	mode_list = list(
 		"Standard" = /datum/lasrifle/base/energy_mg_mode/standard,
-		"Efficiency mode" = /datum/lasrifle/base/energy_mg_mode/standard/efficiency,
-		"Swarm mode" = /datum/lasrifle/base/energy_mg_mode/standard/swarm,
+		"Burst" = /datum/lasrifle/base/energy_mg_mode/standard/burst,
+		"Charge" = /datum/lasrifle/base/energy_mg_mode/standard/charge,
+		"Melting" = /datum/lasrifle/base/energy_mg_mode/standard/melting,
 	)
 
+/obj/item/weapon/gun/energy/lasgun/lasrifle/standard_marine_mlaser/apply_gun_modifiers(obj/projectile/projectile_to_fire, atom/target, firer)
+	. = ..()
+	if(shots_fired) //this specifically boosts the burst fire mode
+		projectile_to_fire.damage *= (1 + shots_fired)
```

Comment:
This seems weird, conceptually. Burst fire already does this by virtue of firing a burst. What's the idea of this separately??

Reply:
?? This is increasing damage per shot fired in a burst

Reply:
Yeah, misunderstood the var, progressive ramp through a burst.